commands/channel.go

```diff
@@ -255,7 +255,7 @@ func addUserToChannel(c client.Client, channel *model.Channel, user *model.User,
 
 func archiveChannelsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 	if len(args) < 1 {
-		return errors.New("Enter at least one channel to archive.")
+		return errors.New("Enter at least one channel to archive")
```

Comment:
Note that the error message has been deliberately modified, otherwise golint throws the following warning.
```
error strings should not be capitalized or end with punctuation or a newline go-lint
```